Sources/ApodiniDatabase/FileHandlers/MultipleDownloader.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import NIO
+import Apodini
+
+/// A Handler which enables the download of files saved to Apodini.
+/// The `MultipleDownloader` will returns all file it finds under the given name or suffix
+/// in the directories specified in the `DownloadConfiguration`.
+/// If you want to download a single files, please refer to `SingleDownloader`
+public struct MultipleDownloader: Handler {
+    @Environment(\.directory)
+    private var directory: Directory
+    
+    @Environment(\.fileio)
+    private var fileio: NonBlockingFileIO
+    
+    @Environment(\.eventLoopGroup)
+    private var eventLoopGroup: EventLoopGroup
```

Comment:
In the future I think it will be better to use the eventLoop of the request

Reply:
I agree! I think for now this is the only way to access an event loop from the handler.